### PR TITLE
Jseldess/env variables

### PR DESCRIPTION
**Pull request by jseldess**
Documented supported env variables.

- Added overview to Cockroach Commands page.
- Listed env variable for relevant flags in docs for specific commands.

As a part of this, I added a new page for the `node` command, but I'll fill out details on that command in a separate PR.

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/docs/235)

### CODE REVIEW

File: cockroach-commands.md [hunk 1]
```diff
@@ -38,4 +36,18 @@ Flag | Description
...
-The `--log-backtrace-at`, `--verbosity`, and `--vmodule` flags are intended for internal debugging. 
\ No newline at end of file
+The `--log-backtrace-at`, `--verbosity`, and `--vmodule` flags are intended for internal debugging. 
+
+## Environment Variables
+
+For many common `cockroach` flags, such as `--port` and `--user`, you can set environment variables once instead of manually passing the flags each time you execute commands. 
+
+- To find out which flags support environment variables, see the documentation for each [command](#commands). 
+- To output the current configuration of CockroachDB environment variables, run `./cockroach debug env`.
```
Comment (bdarnell) [inline cockroach-commands.md]: We should either leave this command undocumented or move it out of `debug`. I would leave it out.

**Comment by bdarnell** [comment]
LGTM

File: cockroach-commands.md (hunk 1 above)
Comment (jseldess) [inline cockroach-commands.md]: Oh, right. I'll remove mention of this for now. Users can just run `env` to see which variables they've set anyway.

**Comment by knz** [comment]
:lgtm_strong: 

---

Reviewed 1 of 1 files at r1.
Review status: 0 of 8 files reviewed at latest revision, 2 unresolved discussions.

---

*[cockroach-commands.md, line 55 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/docs/235#-KFycvOPQXbM3qQ7P9Kj:-KFycvOPQXbM3qQ7P9Kk:-552527858) ([raw file](https://github.com/cockroachdb/docs/blob/e32a7dde666dac2d71fe6a80dc8d4aaee870bafa/cockroach-commands.md#L55)):*
Yes this explanation is correct.

nit: I suggest ending the last line of this doc file with a newline character.

---

*[cockroach-commands.md, line 46 \[r5\]](https://reviewable.io:443/reviews/cockroachdb/docs/235#-KFuC7J1puwb9GAsWwUz-r5-46:-KFycnLnUMQo9cq-L4o9:136721638) ([raw file](https://github.com/cockroachdb/docs/blob/a77ac343c1b655e05d70e0fe0a3d19fb6969b505/cockroach-commands.md#L46)):*
Yes let's not talk about debug too much until we have a clearer story on troubleshooting.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/235#-:-KFyd5kiGbPqFW2M7Urs:-1868698381)*

**Comment by jseldess** [comment]
Review status: 0 of 8 files reviewed at latest revision, 2 unresolved discussions.

---

*[cockroach-commands.md, line 55 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/docs/235#-KFycvOPQXbM3qQ7P9Kj:-KFyg8GS8SQyOtoiHBFI:-499146009) ([raw file](https://github.com/cockroachdb/docs/blob/e32a7dde666dac2d71fe6a80dc8d4aaee870bafa/cockroach-commands.md#L55)):*
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/235)*

**Comment by jseldess** [comment]
Review status: 0 of 8 files reviewed at latest revision, 2 unresolved discussions.

---

*[cockroach-commands.md, line 46 \[r5\]](https://reviewable.io:443/reviews/cockroachdb/docs/235#-KFuC7J1puwb9GAsWwUz-r5-46:-KFyg9tC0Kd2VmgcG24i:-499146009) ([raw file](https://github.com/cockroachdb/docs/blob/a77ac343c1b655e05d70e0fe0a3d19fb6969b505/cockroach-commands.md#L46)):*
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/235)*

**Pull request merged**